### ISSUE TITLE
No notification on write error

### ISSUE THREAD

**Issue by moogaem** [mendhak/gpslogger#1185]
First I want to thank the developer for developing this great app! :-) Now for my problem:

Situation: I restored the latest version app via Neobackup after an upgrade to an higher android version and started the app. It worked as usual (I thought) and in the notification it always said "now" about the time of the last location determination.

Problem: After a week I checked the folder and saw that nothing was written. I checked the folder and a notification came up which said that in this Android version (14) I need to grant extra writing rights which I did then. Now everything is actually working again. So I lost a week of very important tracks. I also checked the log and after every position determination there was a Gpx10FileLogger.write entry in red, which I guess is the writing error due to missing writing rights.

Therefore I would suggest that if there is a write error there should be a notification, since writing is the essential task the logger does.

Otherwise I really like and recommend the app.

**Comment by mendhak** [comment]
Hi which version are you using? I added that feature in v131, have a look at some screenshots in this thread: https://github.com/mendhak/gpslogger/issues/1138

**Comment by moogaem** [comment]
Yes you where right: for some reasons I was using a previous version ... so it was all my fault. Thanks again for this great app! :-)
